src/test/java/bio/terra/service/snapshotbuilder/SnapshotRequestDaoTest.java

```diff
@@ -66,14 +71,21 @@ private SnapshotAccessRequestModel createRequest(SnapshotAccessRequest snapshotA
     return snapshotRequestDao.create(snapshotAccessRequest, EMAIL);
   }
 
-  private void verifyResponseContents(SnapshotAccessRequestModel response) {
+  private void verifyResponseContents(
```

Comment:
How would you feel about creating a helper that was `verifyResponseContents(SnapshotAccessRequestModel response)` that just called this method with `List.of()`? If there's no overrides, we may be able to remove a lot of empty list boilerplate.

Reply:
I love it

Reply:
I think a better approach is to use varargs here. That way you get the `List.of()` case for free. The code would look like:
```
  private void verifyResponseContents(SnapshotAccessRequestModel response, String ...ignoredFields) {
  ...
    // always ignore id and createdDate which are unique
    List<String> allIgnoredFields = new ArrayList<>(Arrays.asList(ignoredFields));
    allIgnoredFields.add("id");
    allIgnoredFields.add("createdDate");
  ...
  }

  @Test
  void getById() {
    verifyResponseContents(snapshotRequestDao.getById(createRequest().id()));
  }
```